source/project-stories/bucks-cc-beta.erb

```diff
@@ -0,0 +1,210 @@
+---
+title: "Project Story - Buckinghamshire County Council"
+
+header:
+  client: "Buckinghamshire County Council"
+  introduction: "Digitalising key local government services with Buckinghamshire County Council: The Beta phase"
+  image: "/project-stories/bucks-cc-beta/header.jpg"
+  image_description: ""
```

Comment:
Hey @mybits - thanks for pointing this out, have added in an image description for the header.